goss-testing/suites/ncn-upgrade-tests-storage.yaml

```diff
@@ -21,10 +21,17 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 #
+
+# Note: The tests in this suite must be run using the `run_goss_tests` helper
+# function defined in `csm-testing/goss-testing/automated/run-ncn-tests.sh`.
+# This is required due to a temporary values file generated in the function
```

Comment:
I would change this so it shows how to run the test instead of saying to use the function.
```suggestion
# Note: The tests in this suite must be run using an automated script
# that uses the `run_goss_tests` helper function.
# This suite can be run by executing `/opt/cray/tests/install/ncn/automated/ncn-upgrade-tests-storage`.
# This is required due to a temporary values file generated in the `run_goss_tests` function.
```

Reply:
Makes sense to me, I have changed to your suggestion.